procedures/unit-testing-tap.ipf

```diff
@@ -306,12 +306,12 @@ static Function TAP_WriteCase(case_cnt, skipcase)
 
 	if(skipcase)
 		str_ok = "ok"
+		tap_diagnostic = ""
```

Comment:
Why is that change needed?
tap_diagnostic is set in TAP_TestCaseBegin to an empty string.

Reply:
If a test case is marked as SKIP, TAP_TestCaseBegin is not executed because the hooks are nested inside
`if(!s.tap_skipCase || reentry)`
within RunTest. So the old message will be displayed as it is not overwritten.
Should the above line be in RunTest at this position?
```
s.tap_skipCase = 0
if(TAP_IsOutputEnabled())
	s.tap_skipCase = TAP_IsFunctionSkip(s.fullFuncName)
endif
```